processing/src/main/java/io/druid/segment/QueryableIndexStorageAdapter.java

```diff
@@ -193,7 +199,8 @@ public DateTime getMaxIngestedEventTime()
       Interval interval,
       VirtualColumns virtualColumns,
       Granularity gran,
-      boolean descending
+      boolean descending,
+      @Nullable QueryMetrics<?> queryMetrics
```

Comment:
instead of putting `null` everywhere, can you make a backwards compatible method signature and just `@Deprecated` it?